README.md

```diff
@@ -104,6 +108,47 @@ at runtime as an environment variable:
 
 See `giftless/config.py` for some default configuration options. 
 
+
+### Running a local example
+
+First, create a new project on Github or any other platform.
+Here, we create `example-proj-datahub-io`. Add any data file
+to it. The goal is to track this possible large file with 
+git-lfs and use Giftless as the local server. In our example,
+we create a CSV named `research_data_factors.csv`.
+
+Next, start the Giftless server. Go to your local project 
+directory and run:
```

Comment:
In terms of style I find that it is much easier to read an example as a list of steps (ordered or bullets) rather than a paragraph of text. The use of "first" and "next" is also confusing because these are used as filler words and not as actual step orders (next is not the next step after first in this description). In general I recommend rephrasing this section so that you state the goals and general things first, then break down the flow into steps. 

It is also worth breaking the shell code into sections with some explanations in the middle otherwise it becomes a huge blob of commands which is hard to follow.

Reply:
It is a relatively short paragraph. No complex instructions in it and the content may vary. I think users may have no problem following it. I don't think 'first' and 'next' are filler words, they just indicate a logical sequence to be followed

Reply:
I would 👍 @shevron comment on bullet points.